tooling/cli/lib/deploy.js

```diff
@@ -64,13 +75,29 @@ const JOB_QUERY = /* GraphQL */ `
     await generateTranslationFile(OP_DIR, [instance], true);
     // trim translations file down to only keys used in project
     await gatherUsedTranslations();
+    // build project
+    const hasAtoms = await dirHasAtoms(OP_DIR);
+    await buildProject(hasAtoms);
+
+    let atomsScriptHash;
+    let atomsStyleHash;
+    if (hasAtoms) {
+      atomsScriptHash = await getAtomsHash(OP_DIR, 'script');
+      atomsStyleHash = (await getAtomsHash(OP_DIR, 'style')) || atomsScriptHash;
+
+      await compileStyles(OP_DIR, atomsStyleHash);
```

Comment:
The hash here seems to only represent the hash of the `style` file, it does not take into account any tailwind.css changes. That means if you deploy, add a few tailwind classes, and re-deploy, the hash will not change, which will lead to those changes not appearing. I think we'll have to calculate a new hash entirely once we `compileStyles` (these hashes are usually just md5 or sha-1 truncated to X characters)